.travis.yml

```diff
@@ -4,6 +4,9 @@ node_js: 7.6
 matrix:
     fast_finish: true
     include:
+      # format code using clang-format
+      - env: FORMAT_CODE=true
```

Comment:
I think you have a better solution, consider changing to code snippet below

```
env: TEST_SUITE=code_format
```

Reply:
yes, I think it's a better solution,i will update the code